packages/trader/src/App/Components/Elements/market-is-closed-overlay.jsx

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Button } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
+import MarketCountdownTimer from './market-countdown-timer.jsx';
 
-const MarketIsClosedOverlay = ({ onClick }) => (
-    <div className='market-is-closed-overlay'>
-        <p>
-            <Localize i18n_default_text='Market is closed.' />
-        </p>
-        <p>
-            <Localize i18n_default_text='Try Synthetic Indices which simulate real-world market volatility and are open 24/7.' />
-        </p>
-        <Button
-            className='market-is-closed-overlay__button'
-            onClick={onClick}
-            text={localize('Try Synthetic Indices')}
-            primary
-        />
-    </div>
-);
+const MarketIsClosedOverlay = ({ is_eu, is_market_available, onClick }) => {
+    let message = '';
+    let btn_text = '';
+    if (is_eu) {
+        btn_text = localize('View open markets');
+        message = is_market_available
+            ? 'Market is closed. Explore other available trading options.'
+            : 'Market is closed.';
+    } else {
+        btn_text = localize('Try Synthetic Indices');
+        message =
+            'Market is closed. Try Synthetic Indices which simulate real-world market volatility and are open 24/7.';
+    }
+
+    return (
+        <div className='market-is-closed-overlay'>
+            <MarketCountdownTimer is_main_page />
+            <p>
+                <Localize i18n_default_text={message} />
```

Comment:
Can we define the localize where we assigned the string and import the dynamic component here? `<p>{message}</p>`. 

It is hard to keep track of where localize is assigned and not, since `btn_text` has the `localize` built-in.